pkg/csi/cinder/driver.go

```diff
@@ -83,21 +85,30 @@ type DriverOpts struct {
 	ClusterID string
 	Endpoint  string
 	PVCLister v1.PersistentVolumeClaimLister
+
+	WithTopology bool
 }
 
 func NewDriver(o *DriverOpts) *Driver {
 	d := &Driver{
-		name:      driverName,
-		fqVersion: fmt.Sprintf("%s@%s", Version, version.Version),
-		endpoint:  o.Endpoint,
-		clusterID: o.ClusterID,
-		pvcLister: o.PVCLister,
+		name:         driverName,
+		fqVersion:    fmt.Sprintf("%s@%s", Version, version.Version),
+		endpoint:     o.Endpoint,
+		clusterID:    o.ClusterID,
+		withTopology: o.WithTopology,
+		pvcLister:    o.PVCLister,
 	}
 
 	klog.Info("Driver: ", d.name)
 	klog.Info("Driver version: ", d.fqVersion)
 	klog.Info("CSI Spec version: ", specVersion)
 
+	if d.withTopology {
+		klog.Infof("Topology awareness enabled")
+	} else {
+		klog.Info("Topology awareness disabled")
+	}
```

Comment:
I think we can make it simple.
```suggestion
	klog.Infof("Topology awareness: %T", d.withTopology)
```

Reply:
This is taken from the Manila driver:

https://github.com/kubernetes/cloud-provider-openstack/blob/dc8e21cdae5657b420f52a972b7d06661dd246f1/pkg/csi/manila/driver.go#L137-L141

Do I change that also?

Reply:
Done.